config/locales/en.yml

```diff
@@ -41,7 +41,7 @@ en:
         registered_name: Registered name
         percent_total: Percentage total
       user:
-        login: Email or FRN
+        login: Firm Reference Number
```

Comment:
If we can assume everyone who has to log-in here has that information, we're fine. But my idea was that if someone doesn't know what an FRN is, they've got no other context on this page that they can use to figure it out. Hence the expansion.